Mathlib/Combinatorics/SimpleGraph/Connectivity.lean

```diff
@@ -2406,6 +2406,47 @@ theorem set_walk_length_succ_eq (u v : V) (n : ℕ) :
 
 variable (G) [DecidableEq V]
 
+theorem set_walk_length_two_bij_commonNeighbors (u v : V) :
+    Set.BijOn (fun p => p.getVert 1) {p : G.Walk u v | p.length = 2} (G.commonNeighbors u v) := by
```

Comment:
Can you construct this as an equivalence? That would probably be shorter overall, and also an extra result.